osf/management/commands/fix_unclaimed_records_for_preprint_versions.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.apps import apps
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        update_unclaimed_records_for_preprint_versions()
```

Comment:
Command looks good, here are a few suggestions:

* Let's add a dry-run option
* Let's add some logging so during either dry-run or REAL run, we can see progress and outputs
* Optional, for places where exception may happen, we should catch them and skip that round (with error logging of course) so that the rest can continue to run